eng/common/tools.ps1

```diff
@@ -312,9 +312,15 @@ function InitializeVisualStudioMSBuild([bool]$install, [object]$vsRequirements =
     return $global:_MSBuildExe
   }
 
-  $vsMinVersionReqdStr = '16.5'
+  # Minimum VS version to require.
```

Comment:
># Minimum VS version to require. [](start = 2, length = 32)

Shouldn't this be auto-updated by Maestro?

Reply:
Seems like [https://github.com/dotnet/sourcelink/pull/650](https://github.com/dotnet/sourcelink/pull/650) is blocked currently. I'd prefer we resolve that issue and let Maestro update these files.

---
In reply to: [494513458](https://github.com/dotnet/sourcelink/pull/651#discussion_r494513458) [](ancestors = 494513458)

Reply:
It was auto-updated by Darc in this case, in my first commit. And yeah, I pointed at https://github.com/dotnet/sourcelink/pull/650 in my comment on this PR. 😄

Reply:
Rebased on top of the merge, now this PR just has the one commit adding basic arcade-powered-source build configuration.